go.mod

```diff
@@ -203,6 +203,9 @@ replace github.com/ulikunitz/xz => github.com/ulikunitz/xz v0.5.11
 // https://deps.dev/advisory/OSV/GO-2021-0053?from=%2Fgo%2Fgithub.com%252Fgogo%252Fprotobuf%2Fv1.3.1
 replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 
+// Use local kvdb package until new version is tagged.
+replace github.com/lightningnetwork/lnd/kvdb => ./kvdb
```

Comment:
Is this tagged now?

Reply:
No, the following commit in this PR modifies `kvdb/postgres` and will have to be tagged after it's merged. Then it will be safe to remove the redirect. Same goes for sqldb.